### PR TITLE
T5 parts

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Makes new post alien tech parts!
Few more chems in dispensers and sleepers.
Thats it

## Why It's Good For The Game

Gives people that get lucky or smart more things to spend the endless mats on as well as a hard work to get even better parts as they need reagents!

## Changelog
:cl:
add: Past alien tech, you can get t5 parts
add: T5 parts unlock new reagents into chem dispenders and sleepers
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/machinery/chem_dispenser.dm [hunk 1]
```diff
@@ -102,6 +110,10 @@
 		upgrade_reagents2 = sortList(upgrade_reagents2, /proc/cmp_reagents_asc)
 	if(upgrade_reagents3)
 		upgrade_reagents3 = sortList(upgrade_reagents3, /proc/cmp_reagents_asc)
+	if(upgrade_reagents4)
+		upgrade_reagents4 = sortList(upgrade_reagents4, /proc/cmp_reagents_asc)
+	if(upgrade_reagents5)
+		upgrade_reagents5 = sortList(upgrade_reagents4, /proc/cmp_reagents_asc)
```
Comment (ghost) [inline code/modules/reagents/chemistry/machinery/chem_dispenser.dm]: upgrade_reagents5

File: code/modules/reagents/chemistry/machinery/chem_dispenser.dm [hunk 2]
```diff
@@ -510,6 +526,16 @@
 		/datum/reagent/consumable/nothing,
 		/datum/reagent/consumable/peachjuice
 	)
+	upgrade_reagents4 = list(
+		/datum/reagent/drug/mushroomhallucinogen,
+		/datum/reagent/consumable/nothing,
+		/datum/reagent/consumable/peachjuice
+	)
```
Comment (ghost) [inline code/modules/reagents/chemistry/machinery/chem_dispenser.dm]: ?

File: code/modules/reagents/chemistry/machinery/chem_dispenser.dm [hunk 3]
```diff
@@ -79,6 +79,14 @@
 		/datum/reagent/toxin
 	)
 
+	var/list/upgrade_reagents4 = list(
+		/datum/reagent/medicine/charcoal
```
Comment (silicons) [inline code/modules/reagents/chemistry/machinery/chem_dispenser.dm]: no

Comment (Trilbyspaceclone) [reply code/modules/reagents/chemistry/machinery/chem_dispenser.dm]: ?

Comment (Trilbyspaceclone) [reply code/modules/reagents/chemistry/machinery/chem_dispenser.dm]: OH woops

Comment (Trilbyspaceclone) [reply code/modules/reagents/chemistry/machinery/chem_dispenser.dm]: wait whats the issue?

Comment (Trilbyspaceclone) [reply code/modules/reagents/chemistry/machinery/chem_dispenser.dm]: Non-Issue

**Comment by YakumoChen** [comment]
ah yes today I will make printable nigh-infinite self-charging power cells

I think we were considering *removing* tier 4 parts at one point. Who gave you the ok to make an even worse tier?

**Comment by ShadeAware** [comment]
Oooh, I _like_ this...

**Comment by Trilbyspaceclone** [comment]
@YakumoChen 
Citation?
Also, you dont need an "ok" to make a pull request on an open source game

**Comment by YakumoChen** [comment]
you should generally ask maintainers if something is a good idea if its going to be a large, sweeping change (like adding a new tier of parts)

**Comment by Trilbyspaceclone** [comment]
@YakumoChen And your citation of removal of t4 parts?

**Comment by shellspeed1** [comment]
They should probably also be available to engineering. That being said adding t5 parts is a bad idea.

**Comment by silicons** [comment]
honestly pretty against this unless something like rp's particle focus is done as a new system rather than just more abductor tech.

**Comment by Trilbyspaceclone** [comment]
@kevinz000 If I port that and make them use the mats, but move them out of alien tech, will you merge it?

**Comment by silicons** [comment]
pending https://github.com/Citadel-Station-13/Citadel-Station-13/pull/12536 and discussions about balancing/unlocking for this.

**Pull request closed without merging**